File: tests/fixtures/lang_samples/javascript/nested.js
function deeplyNested(x) {
  if (x > 0) {
    for (let i = 0; i < x; i++) {
      if (i % 2 === 0) {
        while (i > 0) {
          try {
            if (i === 5) {
              return i;
            }
          } catch (e) {}
          i--;
        }
      }
    }
  }
  return 0;
}

function shallow(x) {
  return x + 1;
}

File: tests/fixtures/lang_samples/rust/nested.rs
fn deeply_nested(x: i32) -> i32 {
    if x > 0 {
        for i in 0..x {
            if i % 2 == 0 {
                let mut j = i;
                while j > 0 {
                    if j == 5 {
                        return j;
                    }
                    j -= 1;
                }
            }
        }
    }
    0
}

fn shallow(x: i32) -> i32 {
    x + 1
}
